README.md

```diff
@@ -398,9 +421,13 @@ You can develop shared functionality or design in common files. To indicate that
 
 ### Development in `platforms`
 
-In `platforms`, you can safely modify files which are part of the native project structure and do not have a corresponding source located in the `app` directory in the root. For example, `AndroidManifest.xml` and `Info.plist`.
+As an important note, you should edit files, located in `platforms` only in extremely rare cases, as the NativeScript CLI overrides such files during the `prepare <Platform>` operation with the content from `app`.
```

Comment:
```
> **IMPORTANT:** Avoid editing files located in the `platforms` subdirectory because the NativeScript CLI overrides such files during the `prepare <Platform>` operation with the content from `app`.
```